lms/djangoapps/branding/models.py

```diff
@@ -0,0 +1,23 @@
+"""
+Model for branding
+
+This was implemented initially so that the list of course tiles could be
+stored in this model.
+"""
+
+#from django.contrib import admin
+from django.db import models
+
+from config_models.models import ConfigurationModel
+from xmodule_django.models import CourseKeyField
+
+
+class TileConfiguration(ConfigurationModel):
+    """
+        Stores a list of tiles presented on the front page.
+    """
+    site = models.CharField(max_length=32, default='default', blank=False)
+    course_id = CourseKeyField(max_length=128, db_index=True)
```

Comment:
Yeah, I'm not worried about making this consistent everywhere (now). But if there _is_ a constant _already_ defined, we should use it. I didn't know if there was one, more curiosity than anything.